go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/containerd/containerd v1.7.2
-	github.com/containerd/nydus-snapshotter v0.9.0
+	github.com/containerd/nydus-snapshotter v0.0.0-20230717034059-ae06355efe37
```

Comment:
Maybe we can release a nydus-snapshotter version for encrypted image scenario. cc @changweige